testing_and_setup/compass/ocean/global_ocean/WC14/init/config_E3SM_coupling_files.ini

```diff
@@ -0,0 +1,73 @@
+# This is the custom config file specific to this test case.  Any values here
+# will override the defaults in defaults.ini  Users wishing to alter the
+# configuration should modify this file, rather than defaults.ini
+
+[main]
+nprocs = 1
+# the file name of the initial condition, possibly after spin-up
+initial_condition = ../../spin_up/simulation/restarts/restart.0001-01-21_00.00.00.nc
+
+[mesh]
+short_name = ${prefix}${min_res}to${max_res}E${e3sm_version}r${mesh_version}
+long_name = ${prefix}${min_res}to${max_res}kmL${levels}E3SMv${e3sm_version}r${mesh_version}
+prefix = CUSP
+description = MPAS Contiguous United States Plus mesh for E3SM version
+              ${e3sm_version}, with a focused ${min_res}-km resolution
+              around North America and ${levels} vertical levels
+e3sm_version = 2
+mesh_version = 01
+creation_date = autodetect
+min_res = 12
+max_res = 60
```

Comment:
This metadata needs to be updated before this PR gets merged.

Reply:
Updated. Thanks for pointing it out.